experimental/packages/opentelemetry-sdk-metrics-base/src/Instruments.ts

```diff
@@ -0,0 +1,68 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import * as api from '@opentelemetry/api';
+import * as metrics from '@opentelemetry/api-metrics';
+import { Meter } from './Meter';
+
+export enum InstrumentType {
+    COUNTER = 'COUNTER',
+    HISTOGRAM = 'HISTOGRAM',
+    UP_DOWN_COUNTER = 'UP_DOWN_COUNTER',
+    OBSERVABLE_COUNTER = 'OBSERVABLE_COUNTER',
```

Comment:
Should they be called `Asynchronous` instead of Observable ?

Reply:
In the code example below the heading you linked, it is named observable.

Reply:
Also see here: https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/metrics/api.md#asynchronous-updowncounter-creation

Reply:
Right, i'm not sure how i missed it ahah but seems weird to have different names in concept and implementations